### PR TITLE
stop using deprecated alias for push-build

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

File: jobs/ci-kubernetes-bazel-build-1-6.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ if [[ "${rc}" == 0 ]]; then
     echo "Kubernetes version missing; not uploading ci artifacts."
     rc=1
   else
-    bazel run //:ci-artifacts -- "gs://kubernetes-release-dev/bazel/${version}" && rc=$? || rc=$?
+    bazel run //:push-build -- "gs://kubernetes-release-dev/bazel/${version}" && rc=$? || rc=$?
```
Comment (ixdy) [inline jobs/ci-kubernetes-bazel-build-1-6.sh]: I don't think we cherry-picked this to 1.6 (yet). I meant to, though.

Comment (mikedanese) [inline jobs/ci-kubernetes-bazel-build-1-6.sh]: I'll not change 1-6.sh for now. I don't think it's a big deal if we don't cherrypick.

File: jobs/pull-kubernetes-bazel.sh [hunk 2]
```diff
@@ -39,7 +39,7 @@ if [[ "${rc}" == 0 ]]; then
     rc=1
   else
     version="${PULL_NUMBER:-batch}/${PULL_REFS:-}"
-    bazel run //:ci-artifacts -- "gs://kubernetes-release-dev/bazel/${version}" && rc=$? || rc=$?
```
Comment (ixdy) [inline jobs/pull-kubernetes-bazel.sh]: we might need to special-case for the release-1.6 branch though?

Comment (spxtr) [inline jobs/pull-kubernetes-bazel.sh]: https://github.com/kubernetes/kubernetes/pull/46653 ?

Comment (ixdy) [inline jobs/pull-kubernetes-bazel.sh]: yup

**Comment by ixdy** [comment]
it should be safe to merge this now.

**Comment by fejta** [comment]
Also IMO it makes more sense to use `kubetest --build=bazel --stage=FOO` to do this

**Comment by ixdy** [comment]
@fejta that is out of scope for this PR. this PR is simply switching to the non-deprecated target (see https://github.com/kubernetes/kubernetes/blob/d3b740bf28743b3fb781e2f75ebdc5b201c60877/build/root/BUILD.root#L55) and has no other functional changes.

https://github.com/kubernetes/test-infra/pull/2732 is a more reasonable place to direct this feedback.

**Pull request merged**